docs/how-to-guides/training-machine-learning-models/training-models.md

```diff
@@ -28,8 +28,17 @@ an example dataset containing three distinct classes (green, yellow, red), which
 
 Detailed instructions for training the traffic light classifier model can be found **[here](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/traffic_light_classifier/README.md)**.
 
-<!--
+## Training CenterPoint 3D object detection model
 
-Training traffic light detection model and lidar CenterPoint model will be added there.
+The CenterPoint 3D object detection model within the Autoware has been trained using the **[open-mmlab/mmdetection3d](https://github.com/open-mmlab/mmdetection3d)** repository. CenterPoint
+implementation of mmdetection3d uses 10 input features for the PointPillars voxel encoder. However, Autoware employs 9 pillar features for CenterPoint, in accordance with the original research paper. Therefore, we have
```

Comment:
These specific change log should be written in https://github.com/open-mmlab/mmdetection3d